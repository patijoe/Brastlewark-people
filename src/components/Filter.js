import React from 'react';

class Filter extends React.Component{
  render() {
    const {handleFilterName} = this.props;

    return(
      <div className="filterName">
        <label htmlFor="filterName" className="filter__label"> Who are you lookin for?</label>
        <input type="text" id="filterName" onChange={handleFilterName} />
      </div>

    );
  }
}

export default Filter;