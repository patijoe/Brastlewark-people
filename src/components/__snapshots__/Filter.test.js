import React from 'react';
import { create } from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Filter from './Filter';

escribe('Filter component', () => {

  it('matches the snapshot in filter component', () => {
    const component = create(
    <Router>
      <Filter
      />
    </Router>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });

});