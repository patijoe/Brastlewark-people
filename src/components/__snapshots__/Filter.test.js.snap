// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter component matches the snapshot in filter component 1`] = `
<div
  className="filterName"
>
  <label
    className="filter__label"
    htmlFor="filterName"
  >
     Who are you looking for?
  </label>
  <input
    id="filterName"
    type="text"
  />
</div>
`;
